packages/core/src/extension/extension-iframe.ts

```diff
@@ -0,0 +1,42 @@
+import { Cart } from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+import { InvalidExtensionConfigError } from './errors';
+import { Extension } from './extension';
+
+export class ExtensionIframe {
+    private _iframe: HTMLIFrameElement;
+
+    constructor(private _containerId: string, private _extension: Extension, cart: Cart) {
```

Comment:
Why not have _ for cart as well, seems like a private member?

Reply:
Since we only pass`extensionId` and `cartId` to an extension here, should I update it to `cartId` instead?

Reply:
Yes I would say so, less is better :D We can always enrich with more data.

Reply:
Thanks. This area has been updated accordingly.